.github/workflows/npmpublish-beta.yml

```diff
@@ -10,12 +10,17 @@ on:
 
 jobs:
   build:
+    
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
```

Comment:
There is no value in having `matrix` in the publish files since we only want to use one node version

Reply:
okay I changed them to be as before but for node 10.

Reply:
Why node 10?  Node 12 is latest LTS.  That only impacts what version of node the github action is run with, not what version of node is supported on the users end